src/main/java/com/example/parser/utils/FileDownload.java

```diff
@@ -0,0 +1,45 @@
+package com.example.parser.utils;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.*;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+public class FileDownload {
```

Comment:
It must be singleton service or utility class with static methods.

Reply:
done